src/sbml2symbolics.jl

```diff
@@ -0,0 +1,31 @@
+using ModelingToolkit, Symbolics
+
+allowed_funs = Dict(
+    "+" => :+,
+    "-" => :-,
+    "*" => :*,
+    "/" => :/,
+    #TODO add translations of SBML functions to Julia
+)
+
+allowed_sym(x) = haskey(allowed_funs,x) ? allowed_funs[x] : DomainError(x,"Unknown SBML function")
+
+function SBML2Symbolics(x::Math)
+    conv(x::SBML.MathApply) =
+        eval(allowed_sym(x.fn))(conv.(x.args)...)
+    conv(x::SBML.MathIdent) =
+        Symbolics.Variable(Symbol(x.id))
+    conv(x::SBML.MathVal) = x.val
+    conv(x::SBML.MathLambda) =
+        throw(DomainError(x, "can't translate lambdas to symbolics"))
+    conv(x)
+end
+
+
+# SBML.MathApply("*", SBML.Math[
+#     SBML.MathApply("*", SBML.Math[
+#         SBML.MathApply("*", SBML.Math[
+#             SBML.MathIdent("c1"),
+#             SBML.MathIdent("k1")]),
+#         SBML.MathIdent("s1")]),
+#     SBML.MathIdent("s2")])
```

Comment:
please clean up the comment notes. Also please check that all lines have \n on the end (so that the ugly label on the end of the last line disappears :] ). There's a switch for that in VSCode, but it's off by default for whatever bad reason.

Reply:
done (I hope)